test/TokenAuthSpec.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+var expect = chai.expect;
+
+describe('AuthenticationTokensResource', function() {
+  var $httpBackend, authenticationTokensResource;
+
+  beforeEach(module('TokenAuth'));
+
+  beforeEach(module(function($provide) {
+    $provide.service('tokenAuthServer', function() {
+      return function() {
+        return '//foo';
+      };
+    });
+
+    $provide.service('tokenAuthClientUuid', function() {
+      return function() {
+        return 'asdf1234';
+      };
+    });
+  }));
+
+  beforeEach(inject(function(_$httpBackend_, _authenticationTokensResource_) {
+    $httpBackend = _$httpBackend_;
+    authenticationTokensResource = _authenticationTokensResource_;
+  }));
+
+  afterEach(function() {
+    $httpBackend.verifyNoOutstandingExpectation();
+    $httpBackend.verifyNoOutstandingRequest();
+  });
+
+  describe('#create', function() {
+    it('provides credentials to the server', function() {
+      var data = { data: { type: 'authenticationTokens', clientUuid: 'asdf1234' } };
+      $httpBackend.expectPOST('//foo/api/authentication_tokens' +
+                              '?configurationToken=my-token', data)
+                  .respond();
+      authenticationTokensResource.create('my-token');
+      $httpBackend.flush();
+    });
```

Comment:
Do you need an `expect` here?  I see in examples of `expectPOST` that there seems to be more of a clear response or expectation: https://docs.angularjs.org/api/ngMock/service/$httpBackend.